.github/workflows/main_cd.yml

```
@@ -7,15 +7,17 @@ on:
-  DOTNET_VERSION: '9.0.x' # The .NET SDK version to use
+  DOTNET_VERSION: '10.0.x' # The .NET SDK version to use
 
 jobs:
   release-nuget:
     runs-on: windows-latest
 
     steps:
       - uses: actions/checkout@v4
-
+        with:
+          dotnet-version: '10.0.x'
```

actionlint (1.7.8)

input "dotnet-version" is not defined in action "actions/checkout@v4". available inputs are "clean", "fetch-depth", "fetch-tags", "filter", "github-server-url", "lfs", "path", "persist-credentials", "ref", "repository", "set-safe-directory", "show-progress", "sparse-checkout", "sparse-checkout-cone-mode", "ssh-key", "ssh-known-hosts", "ssh-strict", "ssh-user", "submodules", "token"

(action)